README.md

```diff
@@ -67,6 +67,48 @@ Complete the following steps before you deploy the Secure Landing Zone module.
 
     Select **All Identity and Access enabled services** when you assign access to the group.
 
+### Steps to prevent Destroy and recreation of cluster.
+1. Prior to `terraform plan` and `terraform apply`. Edit `moved_config.tf`
+2. Add the following lines to the terraform file.
+  ```
+  moved {
+    from = ibm_container_vpc_cluster.cluster["<workload_cluster_name>"]
+    to   = module.cluster["<workload_cluster_name>"].ibm_container_vpc_cluster.cluster[0]
+  }
+  moved {
+    from = ibm_container_vpc_cluster.cluster["<management_cluster_name>"]
+    to   = module.cluster["<management_cluster_name>"].ibm_container_vpc_cluster.cluster[0]
+  }
+  ```
+3. Make sure to replace the `<workload_cluster_name>` with the actual workload cluster name. Similarly `<management_cluster_name>` with the management cluster name.
+4. Go ahead with `terraform plan` and `terraform apply`.
+
+#### Alternative approach using terraform cli:
+1. Run this command to retrive the cluster names from the state file.
+```
+  terraform state list | grep "ibm_container_vpc_cluster.cluster"
+```
+2. Take the cluster name from the list. For example: `debug-ocp-management-cluster`, `debug-ocp-workload-cluster`
+3. Run `terraform init` to initialize newly added resources.
+4. For each element in the list:
+```
+terraform state mv 'module.landing_zone.ibm_container_vpc_cluster.cluster["<cluster name>"]' 'module.landing_zone.module.cluster["<cluster name>"].ibm_container_vpc_cluster.cluster[0]'
+```
+5. Run `terraform apply` to apply the changes to the infrastructure without re-creating cluster.
+
+6. (Optional) If you have additional worker pools other than the default pool. You will need to move them as well to prevent destroy and recreate. Similiar to the previous steps run the following.
+```
+1. terraform state list | grep "ibm_container_vpc_worker_pool.pool"
+module.landing_zone.ibm_container_vpc_worker_pool.pool["debug-ocp-workload-cluster-logging-worker-pool"]
+
+2. Seperate the cluster name and the worker pool name. For example:
+cluster_name : debug-ocp-workload-cluster
+worker_pool_name : logging-worker-pool
+
+3. For each element in the list, Run the following:
+terraform state mv 'module.landing_zone.ibm_container_vpc_worker_pool.pool["<cluster_name>-<worker_pool_name>"]' 'module.landing_zone.module.cluster["<cluster_name>"].ibm_container_vpc_worker_pool.pool["<worker_pool_name>"]'
+```
```

Comment:
If it's a set of commands to run, introduce each (or the set), but don't put numbers with the commands. Like, 

1. Get the list of Terraform state files by running this command: 
    ```tf
    terraform state list | grep "ibm_container_vpc_worker_pool.pool" module.landing_zone.ibm_container_vpc_worker_pool.pool["debug-ocp-workload-cluster-logging-worker-pool"]
    ```

1.  Locate the cluster and worker pool names. (not sure why you're doing this. can you provide more context?)
1.  Run the following command for each element in the list